services/filemanager/s3manager.go

```diff
@@ -192,6 +192,7 @@ func (manager *S3Manager) getSession(ctx context.Context) (*session.Session, err
 		}
 		manager.Config.Region = aws.String(region)
 	}
+
 	if manager.Config.Region == nil {
```

Comment:
Is this block needed? Will manager.Config.Region ever be nil?

Reply:
Make sense, it is not needed, we can remove it.

Reply:
Removed it